src/WebJobs.Script/Metrics/MetricConstants.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Metrics
+{
+    public class MetricConstants
+    {
+        // Meters
+        public const string Host = "Host";
+
+        // Metrics
+        public const string AppFailureCount = "host.app-failure-count";
```

Comment:
OTel advice is to use snake case for within a section -> `host.app_failure_count`. But they also recommend pluralization and leaving the `count` off of counter values, so `host.app_failures`.

But then they also advise to have the namespace considered universally, so all your metrics should start with someone distinct to your scenario. In this case `host` is a bit generic. We would need something like `azure.functions.host.app_failures` or `azfunc.host.app_failures`.

But _then_ there is already the `faas` semantic conventions. Does this just fit with that? (See other comment). If not, do we add our own namespace to `faas`? So `faas.azfunc.host.app_failures` (or something like that).